pkg/apis/navigator/validation/elasticsearch.go

```diff
@@ -102,6 +103,32 @@ func ValidateElasticsearchCluster(esc *navigator.ElasticsearchCluster) field.Err
 	return allErrs
 }
 
+func ValidateElasticsearchClusterUpdate(old, new *navigator.ElasticsearchCluster) field.ErrorList {
+	allErrs := ValidateElasticsearchCluster(new)
+
+	fldPath := field.NewPath("spec")
+
+	npPath := fldPath.Child("nodePools")
+	for i, newNp := range new.Spec.NodePools {
+		idxPath := npPath.Index(i)
+
+		for _, oldNp := range old.Spec.NodePools {
+			if newNp.Name == oldNp.Name {
+				if !reflect.DeepEqual(newNp.NodeSelector, oldNp.NodeSelector) {
```

Comment:
Is reflect.DeepEqual usually used for Update validations? If so, 👍

Reply:
Yep, that's what's used upstream